src/pages/payment-methods/_expanded-list.tsx

```diff
@@ -103,24 +109,38 @@ const Withdrawal = styled(Td)`
         max-width: 14rem;
     }
 `
-// const replaceLineBreak = (str) => {
-//     return str.toString().replace(/\n/g, '</br>')
-// }
+type ExpandListProps = {
+    data?: {
+        method?: string | ReactElement
+        currencies?: string | ReactElement
+        min_max_deposit?: string | ReactElement
+        min_max_withdrawal?: string | ReactElement
+        deposit_time?: string | ReactElement
+        withdrawal_time?: string | ReactElement
+        description?: string | ReactElement
+        name?: string
+        reference?: string
+        locales?: string[]
+        url?: string
+        reference_link?: string | ReactElement
+    }
+    is_fiat_onramp: boolean
+    locale?: {
+        locale?: { language: string }
+    }
+}
 
-const ExpandList = ({ data, /*config,*/ is_crypto, is_fiat_onramp, locale }) => {
+const ExpandList = ({ data, is_fiat_onramp, locale }: ExpandListProps) => {
     const [is_expanded, setIsExpanded] = React.useState(false)
     const toggleExpand = () => {
         setIsExpanded(!is_expanded)
     }
 
-    // const getCryptoConfig = (name) => {
-    //     return config == undefined ? null : getCryptoDecimals(config[name].minimum_withdrawal)
-    // }
     return (
         <>
             <Tr is_expanded={is_expanded}>
                 <Td>{data.method}</Td>
-                <Td colSpan={is_fiat_onramp && '2'}>
+                <Td colSpan={is_fiat_onramp && parseInt('2')}>
```

Comment:
Let's assign `parseInt('2')}` in to a variable.

Reply:
okay